src/chart/bar/BarView.js

```diff
@@ -127,15 +128,50 @@ export default echarts.extendChartView({
 
         var roundCap = seriesModel.get('roundCap', true);
 
+        var backgroundModel = seriesModel.getModel('backgroundStyle');
+        var backgroundColor = backgroundModel.get('color');
+
+        // Not create background element for transparent one to improve efficiency
+        var drawBackground = backgroundColor !== 'transparent' && backgroundColor !== 'none';
+
```

Comment:
Only using `backgroundColor` to determine if drawing background is not enough. Developers may want to have border only background.

We can use `showBackground ` as mentioned in the previous comment.